### PR TITLE
Clean UnderlyingConnection on Dispose

### PR DESCRIPTION
**Pull request by AlexUstinov**
This is just to make sure connection is not somehow being hold after session Dispose.

### CODE REVIEW

File: Orm/Xtensive.Orm/Sql/SqlConnection.cs [hunk 1]
```diff
@@ -67,28 +69,34 @@ public ConnectionInfo ConnectionInfo
...
     /// </summary>
     /// <returns>Created command.</returns>
     public DbCommand CreateCommand()
     {
+      EnsureIsNotDisposed();
       var command = CreateNativeCommand();
-      if (commandTimeout!=null)
+      if (commandTimeout!=null) {
         command.CommandTimeout = commandTimeout.Value;
+      }
```
Comment (alex-kulakov) [inline Orm/Xtensive.Orm/Sql/SqlConnection.cs:98]: Strangely, you still have editorconfig settings not applied sometimes

File: Orm/Xtensive.Orm/Sql/SqlConnection.cs [hunk 2]
```diff
@@ -267,77 +278,85 @@ public virtual void Rollback()
...
-    /// Ensures the trasaction is not active (i.e. <see cref="ActiveTransaction"/> is <see langword="null"/>).
+    /// Ensures the transaction is not active (i.e. <see cref="ActiveTransaction"/> is <see langword="null"/>).
     /// </summary>
-    protected void EnsureTrasactionIsNotActive()
+    protected void EnsureTransactionIsNotActive()
     {
-      if (ActiveTransaction!=null)
+      if (ActiveTransaction!=null) {
         throw new InvalidOperationException(Strings.ExTransactionShouldNotBeActive);
+      }
```
Comment (alex-kulakov) [inline Orm/Xtensive.Orm/Sql/SqlConnection.cs:357]: Whitespaces absent

File: Orm/Xtensive.Orm/Sql/SqlConnection.cs (hunk 1 above)
Comment (AlexUstinov) [reply Orm/Xtensive.Orm/Sql/SqlConnection.cs]: Fixed

File: Orm/Xtensive.Orm/Sql/SqlConnection.cs (hunk 2 above)
Comment (AlexUstinov) [reply Orm/Xtensive.Orm/Sql/SqlConnection.cs]: Fixed

**Review by alex-kulakov — DISMISSED** [review]

**Review by alex-kulakov — APPROVED** [review]

**Pull request merged**